doc/DESIGN_AND_FEATURES.md

```diff
@@ -56,12 +56,14 @@ The API Server works as follows:
 | `/api/query/session/disabledValidators` | `websocket` | None | List of `disabled validators` |
 | `/api/query/session/validators` | `websocket` | None | List of `validators` |
 | `/api/query/staking/activeEra` | `websocket` | None | `index` and `start` of the `active era` |
+| `/api/query/staking/bonded` | `websocket`, `account_id` | None | Address of controller assigned to account(stash) |
```

Comment:
The description should be the following `Map from all locked "stash" accounts to the controller account.`